state/postgres_version.go

```diff
@@ -13,8 +10,8 @@ const (
 	PostgresVersion14 = 140000
 	PostgresVersion15 = 150000
 
-	// MinRequiredPostgresVersion - We require PostgreSQL 9.3 or newer
-	MinRequiredPostgresVersion = PostgresVersion93
+	// MinRequiredPostgresVersion - We require PostgreSQL 9.6 or newer
+	MinRequiredPostgresVersion = PostgresVersion96
```

Comment:
There is a hard-coded "9.3 or newer is required." message on the top of `input/full.go` that also needs to be updated.

Reply:
Re-using this thread - On another note, I just realized we should probably raise this to Postgres 10 being the minimum - because that was the first version to use $n style replacement characters in pg_stat_statements.

See https://github.com/pganalyze/collector/pull/383, which will effectively make Postgres 9.6 not work with the collector (since we fail to parse / normalize the statements containing "?").

Reply:
Makes sense. I'll update once #339 lands.